api/src/common/exceptions.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self,
         message: str = "The requested resource could not be found",
         debug: str = "The requested resource could not be found",
-        extra: dict = None,
+        extra: dict | None = None,
```

Comment:
For den uvitende ingeniør,  hvorfor er dette er en forbedring? 

Skjønner ikke hvordan jeg skal google meg fram til det 🤓

Reply:
<sup>I think lingua franca for Equinor git repos is English.</sup>

I just opened the repository in VSCode and turned on type checking, and it warned me that `None is not an instance of dict`. Basically `None` is not a valid value when the type is `dict`. This is to make sure that whenever we _do_  type something only as `dict`, we can be sure it will exist (and not be `None`), so that we can e.g. do `myVar.get("key", "defaultValue")` on it.

Reply:
aha, that makes sense. Thanks!